addon/mixins/hot-component.js

```diff
@@ -5,10 +5,12 @@ export default Ember.Mixin.create({
   init () {
```

Comment:
We used to have two classes inheriting from this one at some point, now that there is only one I'll inline the mixin.